index.json

```
@@ -93,7 +93,44 @@
       "repo": "https://github.com/cfengine/masterfiles",
       "by": "https://github.com/cfengine",
       "commit": "c39b79c0e7a42522c69cff3d87a5bc5ac9471369",
-      "steps": ["run ./autogen.sh", "delete ./autogen.sh", "copy ./ ./"]
+      "steps": [
+          "run ./autogen.sh",
+          "delete ./autogen.sh",
+          "delete ./config.guess",
+          "delete ./config.log",
+          "delete ./config.status",
+          "delete ./config.sub",
+          "delete ./configure",
+          "delete ./configure.ac",
+          "delete ./example_def.json",
+          "delete ./install-sh",
+          "delete ./Makefile",
+          "delete ./Makefile.am",
+          "delete ./Makefile.in",
+          "delete ./aclocal.m4",
+          "delete ./missing",
+          "delete ./promises.cf.in",
+          "delete ./standalone_self_upgrade.cf.in",
+          "delete ./autom4te.cache",
+          "delete ./tests",
+          "delete ./m4",
+          "delete ./travis-scripts",
+          "delete ./test-driver",
+          "delete ./CFVERSION",
+          "delete ./README.md",
+          "delete ./CODE_OF_CONDUCT.org",
+          "delete ./CHANGELOG.md",
+          "delete ./CONTRIBUTING.md",
+          "delete ./LICENSE",
+          "delete ./MPF.md",
+          "delete ./.CFVERSION",
+          "delete ./.gitignore",
+          "delete ./.mailmap",
+          "delete ./.git",
+          "delete ./.travis.yml",
+          "delete ./controls/update_def.cf.in",
```

As indicated in slack, I think this list should be in the masterfiles repo, not in the index / cfbs json. We can put a `cleanup.sh` script in masterfiles:

```suggestion
          "run cleanup.sh",
          "delete cleanup.sh",
```